examples/signal.php

```diff
@@ -4,6 +4,10 @@
 
 use Castor\Attribute\AsTask;
 
+if (!\defined('SIGUSR2')) {
+    \define('SIGUSR2', 12);
```

Comment:
not defined in the static binary with the defaults options `mbstring,phar,posix,tokenizer`